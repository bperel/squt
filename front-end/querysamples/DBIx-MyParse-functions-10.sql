SELECT GET_FORMAT(DATE,'INTERNAL')
