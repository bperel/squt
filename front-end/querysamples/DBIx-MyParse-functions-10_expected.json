{
  "Warning": {
    "No alias": {
      "get_format": ["function", "SELECT"]
    }
  },
  "Functions": {
    "0": {
      "to": "OUTPUT",
      "Constants": {
        "INTERNAL": "INTERNAL"
      },
      "name": "get_format",
      "alias": "0"
    }
  }
}